### ISSUE TITLE
Title Tag Ordering

### ISSUE THREAD

**Issue by mcmullengreg** [XavierUniversity/V4#29]
SEO practices dictate ordering should be Page Level - Department Level - Organization Level for `title` ordering. 

We should have a discussion whether we reorder page titles (we could potentially pull this info automatically / aside from page title, similar to the breadcrumbs).

The question is: Do we want to flip the order?

**Comment by lieslandr** [comment]
Some initial thoughts. . . 

This is a hard one for me, but I think it makes sense to flip the order. The favicon will continue to let users know they are on our site. 

I want to think more about whether or not we should automate it. Maybe automate it when possible, if the Campus Suite title is blank. That could lead to some long titles. We could also just pull in the page title, the main dept title, and then the general XU title.

When we are ready to make this change, I could have a student write a script to update current page titles, maybe folder by folder.

**Comment by mcmullengreg** [comment]
Makes sense. Best part of it would be we can tack on - Xavier University at the end of any page, regardless of what they write, automating what we can. 

Personally, I believe we should automate whatever we can, but keep in mind length. Although, I wouldn't worry too much about it since relevance-wise the page name will be the most important in search and/or browser tabs.

**Comment by mcmullengreg** [comment]
We'll be automating this process. `PageTitle` will ONLY be the name of the page instead of listing out the entirety of the title string.
